### PR TITLE
Add Patrick to People pages

### PR DESCRIPTION
**Pull request by pkalita-lbl**
This is mainly giving myself a people page. 

A couple of other small changes in here:
* Added a `Gemfile` and setup instructions using Bundler. This should reduce the overhead for people running this project for the first time in the future and keep development environments consistent
* Updated the people page template to limit the maximum headshot size and to not show missing links

### CODE REVIEW

**Review by kltm — COMMENTED** [review]
Hm. Is Ruby 3.0 really necessary for this? Unfortunately, this setup knocked the wheels off of my ruby installation--I basically get permanently hung up on a rather annoying nokogiri issue that seems to be going around, where it is unable to determine my architecture:
```
ERROR:  While executing gem ... (Gem::RemoteFetcher::FetchError)
    bad response Forbidden 403 (https://api.rubygems.org/quick/Marshal.4.8/nokogiri-1.13.3-x64-unknown.gemspec.rz)
```
Not really the fault of this setup (this doesn't even use `nokogiri`), but annoying that I can't just add this in. Separately, I wonder about the `PLATFORMS` variable (although nothing I did, including playing with that, was able to clear my issue).

I had better luck in a docker container:
```
docker run --network host -it ruby:3.0 /bin/bash
cd && git clone https://github.com/berkeleybop/berkeleybop.github.io.git && cd berkeleybop.github.io
bundle install && bundle exec jekyll serve --livereload
```
It all seemed fine and the changes fine from there.

So, in summary:

* not wild about the environment/build changes, which now seem incompatible for my setup, but a workaround was found that should be added to the instructions if the environment/build is not revised
* everything else seems fine and as advertised

**Comment by pkalita-lbl** [comment]
Oh hmmmm, I'm not really a Ruby expert. I was just sort of aware that Bundler was the equivalent in the Ruby world to Python's virtual environments or Node's `npm` command, which I'm more familiar with. Since I was starting from scratch it seemed to make more sense to use that instead of installing Jekyll globally (which I think is what the existing `README` assumed). I don't really know what `nokogiri` is or what the cause of that problem might be (see first statement about not being a Ruby expert 😁). But I'm happy to rollback my `README` and `Gemfile` changes.

**Comment by kltm** [comment]
No worries--I'm not sure what's "right" in this case, just that it didn't work for me after poking it a little. My ruby environment has gone through a lot of iterations through the years, so it could well just me have quirks. Anyways, we can revisit that separately maybe from getting you onto the website.

**Comment by kltm** [comment]
With that, I'll go ahead and take the PR.

**Comment by pkalita-lbl** [comment]
Yeah that totally makes sense. Thanks for merging!

**Comment by nlharris** [comment]
great, your new page shows up!

do you remember if anyone had a particularly large photo on their page, so we can see whether your size limitation worked?

**Comment by pkalita-lbl** [comment]
Yep, here's the actual file: https://raw.githubusercontent.com/berkeleybop/berkeleybop.github.io/master/people/patrick-kalita/patrick-kalita.jpg. It's 600x600px and it looks like it is correctly scaled to a maximum width of 350px on the page here https://berkeleybop.github.io/people/patrick-kalita/
